packages/nuxt/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@clerk/nuxt",
+  "version": "0.0.0",
+  "description": "Clerk SDK for Nuxt",
+  "keywords": [
+    "clerk",
+    "typescript",
+    "vue",
+    "nuxt",
+    "auth",
+    "authentication",
+    "passwordless",
+    "session",
+    "jwt"
+  ],
+  "homepage": "https://clerk.com/",
+  "bugs": {
+    "url": "https://github.com/clerk/javascript/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/clerk/javascript.git",
+    "directory": "packages/nuxt"
+  },
+  "license": "MIT",
+  "author": "Clerk",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/module.d.ts",
+      "import": "./dist/module.js"
+    },
+    "./server": {
+      "types": "./dist/runtime/server/index.d.ts",
+      "import": "./dist/runtime/server/index.js"
+    }
+  },
+  "main": "./dist/module.js",
+  "types": "./dist/module.d.ts",
+  "files": [
+    "dist",
+    "server"
+  ],
+  "scripts": {
+    "build": "tsup",
```

Comment:
Can you also add a `"dev": "tsup --watch"` script?

Reply:
Added to both vue and nuxt package. thanks!